templates/institutional/institution-arms-en.html

```diff
@@ -16,15 +16,15 @@
 ---
 <h1 property="name" id="wb-cont">{{ page.title }}</h1>
 <div class="row">
-	<div class="col-sm-5 col-xs-12 pull-right gc-rms-lngth">
+	<div class="col-sm-5 col-xs-12 pull-right gc-rms-lngth mrgn-bttm-lg">
```

Comment:
There was no `mrgn-bttm-lg` in the orignal code

```suggestion
	<div class="col-sm-5 col-xs-12 pull-right gc-rms-lngth">
```

I can understand it might need it but that change are out of scope of this PR because it will most likely trigger a minor change for this institutional template according to our versioning API